website/content/docs/configuration/seal/seal-ha.mdx

```diff
@@ -0,0 +1,105 @@
+---
+layout: docs
+page_title: Seal High Availability (Beta) - Seals - Configuration
+description: |-
+  How to configure multiple Seals for high availability.
+---
+
+# Seal High Availability (Beta)
+
+<Note title="Seal HA functionality requires Vault Enterprise">
+
+  Seal High Availability requires Vault Enterprise.
+
+</Note>
+
+-> **Warning:** This feature is available as a Beta for evaluation and should not 
+be used in production deployments of Vault.
+
+[Seal High Availability](/vault/docs/content/concepts/seal#Seal-High-Availability) 
```

Comment:
This link does not work, change it to /vault/docs/concepts/seal#seal-high-availability-enterprise-beta may be?